contracts/src/SubnetRegistryDiamond.sol

```diff
@@ -68,6 +71,8 @@ contract SubnetRegistryDiamond {
         s.subnetActorRewarderSelectors = params.subnetActorRewarderSelectors;
         s.subnetActorCheckpointerSelectors = params.subnetActorCheckpointerSelectors;
         s.subnetActorPauserSelectors = params.subnetActorPauserSelectors;
+
+        s.permissionMode = params.permissionMode;
```

Comment:
`permissionMode` is an overloaded term. We use it to denote the power allocation mode too (collateral, federated, static). It's going to cause ambiguity. Let's change this to `s.allowedCreators` here and everywhere else (enum type name, etc.).